lib/appscale.py

```diff
@@ -100,6 +107,8 @@ class AppScale():
   undeploy <appid>                  Removes <appid> from the current
                                     deployment. DATA ASSOCIATED WITH
                                     THE APPLICATION WILL BE LOST.
+  upgrade                           Upgrades AppScale code to its latest version
```

Comment:
I'd change the description to: 

> Upgrades AppScale to its latest version.

The data upgrade will probably not be a regular op and is part of the whole process.